icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java

```diff
@@ -489,6 +489,45 @@ public void notationCompact() {
                 "Kun");
     }
 
+    @Test
+    public void unitUMinLocale() {
+        assertFormatSingleMeasure(
```

Comment:
Should test all combinations of:
2 alternates for the input measure: Fahrenheit and celsius
3 alternates for usage: none, "default", and "other".
2 alternates for base locale: en, en_AU, and de
each combination of +/- mu, +/- ms and +/- rg, eg 8 combinations (including none).

Easiest if driven from array of test cases + expected output

Reply:
Done.

Reply:
@macchiati I see what looks like a good set of test cases but not the full set of all combinations that you seemed to ask for. Are you satisfied?

Reply:
For the test cases:
-- 2 alternates for the input measure: Fahrenheit and celsius
**I have added them**

-- 3 alternates for usage: none, "default", and "other".
**I have used `none` and `default`. `Other` will give the same result as default, I did not tested it.**

-- 2 alternates for base locale: en, en_AU, and de
**I have used en_US for the test cases, shall I test other locales with different languages?  it will be robust, but I think the main outcome of the test cases that we need is to test the effect of the tags.**

Reply:
@macchiati ok?

Reply:
Mark says test cases are ok now.